edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Program.cs

```diff
@@ -118,6 +118,13 @@ public static async Task<int> MainAsync(IConfigurationRoot configuration)
             ConfigUpdater configUpdater = await container.Resolve<Task<ConfigUpdater>>();
             await configUpdater.Init(configSource);
 
+            if (!Enum.TryParse(configuration.GetValue("AuthenticationMode", string.Empty), true, out AuthenticationMode authenticationMode)
```

Comment:
Should this be case insensitive?

Reply:
It is, the second parameter "true" is for ignoreCase

Reply:
Ah I see.